### PR TITLE
fix: txAdmin Heal Everyone functionality

### LINKED ISSUE #94: When player dead cant be revive by txAdmin
### Summary

You cant revive all player in txAdmin

### Reproduction

I cant revive all in txAdmin and there will be have an error I dont know why this occured
See error below
error

### Expected behavior

Could revive all in txadmin

### Actual behavior

Error happend and player still be dead, I need to selected the player to rivive

### Additional context

_No response_

### Current Version

V1.0.0

### Custom Resources

No

### PR DESCRIPTION
**Pull request by mafewtm**
## Description

Fixes #94 though I feel these events might be exploitable 

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -39,26 +39,23 @@ end)
...
+    TriggerClientEvent('qbx_medical:client:playerRevived', player --[[@as number]])
 end
 
 exports('Revive', revivePlayer)
 
 ---removes all ailments, sets to full health, and fills up hunger and thirst.
 ---@param src Source
 local function heal(src)
-	lib.callback.await('qbx_medical:client:heal', src, 'full')
+    TriggerClientEvent('qbx_medical:client:heal', src, 'partial')
```
Comment (Manason) [inline server/main.lua:50]: Why the change from a full to a partial heal?

**Review by Manason — CHANGES_REQUESTED** [review]
How does changing this from a callback to an event fix txAdmin?

File: server/main.lua (hunk 1 above)
Comment (mafewtm) [reply server/main.lua:50]: just a typo

**Comment by mafewtm** [comment]
> How does changing this from a callback to an event fix txAdmin?

Currently the event handler, [`txAdmin:events:healedPlayer`](https://github.com/Qbox-project/qbx_medical/blob/main/server/main.lua#L71) passes an id which is either the player id or -1 and then passes that to the `revivePlayer` and `heal` functions. Inside these functions are the ox_lib callbacks which can't take -1 as a source. This then breaks the ability to heal everyone through txAdmin hence why I've now changed it to events

**Review by Manason — APPROVED** [review]

**Pull request merged**